samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -111,28 +87,51 @@ private void OnEnable()
         }
     }
 
-    private void PopulateSampleSceneList()
+    public void SetPipelineText(string text)
     {
-        SceneDropdown.options.Clear();
+        PipelineModeText = text;
+    }
 
-#if !(USE_OPENXR_PACKAGE)
-        if (SceneList.Contains("VR-Sample")){ 
-             SceneList.Remove("VR-Sample");
-        }
-#else
-        if (!SceneList.Contains("VR-Sample")){ 
-             SceneList.Add("VR-Sample");
-        }
-#endif
-        SceneDropdown.AddOptions(SceneList);
-        AddScene();
+    public void SetSceneText(string text)
+    {
+        SceneText = text;
     }
 
     private void AddScene()
     {
-        SceneName = SceneDropdown.options[SceneDropdown.value].text;
+        SceneName = SceneText;
+
         //The scene must also be added to the build settings list of scenes
-        SceneManager.LoadSceneAsync(SceneName, new LoadSceneParameters(LoadSceneMode.Additive));   
+        SceneManager.LoadSceneAsync(SceneName, new LoadSceneParameters(LoadSceneMode.Additive));
+    }
+
+    // Switch scenes with button click
+    public void SceneButtonOnClick() {
+        
+        StopVideo();
+
+        cam.enabled = false;
+
+        animator.Play("NotificationAnim_Close");
+
+        // If no async scene is running, then just load an async scene
+        if (SceneLoadedCount == 1)
+        {
+            AddScene();
+        }
+
+        // If there is an async scene running, unload the current scene and load a new async scene
+        else if (SceneLoadedCount == 2)
+        {
+            // Change scene
+            var DoneUnLoadingOperation = SceneManager.UnloadSceneAsync(SceneName);
```

Comment:
`Unloading` is one word
```suggestion
            var DoneUnloadingOperation = SceneManager.UnloadSceneAsync(SceneName);
```